packages/plugin-ext/src/main/browser/menus/menus-contribution-handler.ts

```diff
@@ -496,21 +497,27 @@ export class MenusContributionPointHandler {
             return Disposable.NULL;
         }
 
-        const selectedResource = () => {
+        const getSelectedResources = () => {
             const selection = this.selectionService.selection;
+            const resourceParams = [];
             if (TreeWidgetSelection.is(selection) && selection.source instanceof TreeViewWidget && selection[0]) {
-                return selection.source.toTreeViewSelection(selection[0]);
+                resourceParams[0] = selection.source.toTreeViewSelection(selection[0]);
+            } else {
+                const uri = this.resourceContextKey.get();
+                resourceParams[0] = uri ? uri['codeUri'] : undefined;
             }
-            const uri = this.resourceContextKey.get();
-            return uri ? uri['codeUri'] : undefined;
+            if (TreeWidgetSelection.is(selection) && selection.source instanceof FileTreeWidget && selection.length > 1) {
+                resourceParams.push(selection.map((item: any) => item.uri && item.uri.codeUri));
```

Comment:
This seems to work, but why can we assume that the selection items always contain a `uri`? Can we add an `in` check for the uri? This seems to be quite unsafe to me.